docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -88,13 +87,16 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 ## Options
 
 ### Job Type
+
 #### --interactive
 
 > Mark this Job as interactive.
 
+<!--
```

Comment:
why do you comment-out rather than remove? we don't plan to bring it back, and even if we do, git keeps the history, so it's recommended not to keep dead code.

Reply:
I always try to comment out until I get approval that this is what needs to be deleted. Since you asked the question, I will delete it before I merge.